src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -132,11 +132,30 @@ public Test BuildFrom(FixtureMethod method, Test suite)
                     tests.Add(test); 
             }
 
-            return hasBuildersSpecified && method.Method.GetParameters().Length > 0 || tests.Count > 0
+            return hasBuildersSpecified && method.Method.GetParameters().Length > 0 || 
+                   tests.Count > 0 ||
+                   HasMethodGotDataAttributeParameters(method)
```

Comment:
@DaiMichael I have a couple of ideas. Would you mind either:

 - Getting rid of the colloquialism ‘got,’ maybe using `MethodHasDataAttributeParameters`
 - Or, adding `using System.Linq;` and using `method.Method.GetParameters().Any(p => p.HasAttribute<DataAttribute>())` for this line.

Either one is good by me. I really appreciate it!